pkg/services/login/authinfo.go

```diff
@@ -8,4 +8,5 @@ import (
 
 type AuthInfoService interface {
 	LookupAndUpdate(ctx context.Context, query *models.GetUserByAuthInfoQuery) (*models.User, error)
+	GetAuthInfo(ctx context.Context, query *models.GetAuthInfoQuery) error
```

Comment:
I add it it here because `http_server` is using this function to avoid to add the store dependency.